setup.py

```diff
@@ -61,6 +61,7 @@ def clean_scheme(version):
         'test': [
             'tblib',
             'wurlitzer',
+            'vcrpy',
```

Comment:
Is  `vcrpy` actually in use in the final tests? Looks like this might not be necessary yet - or I just missed it.

Reply:
Nevermind. I was looking at a stale version of the code.

Reply:
It is necessary when using `--load-http-traffic` or `--save-http-traffic` that is why I put it in `test`. But the feature is optional that is why in test but not a requirement.